src/coreclr/src/jit/compiler.h

```diff
@@ -9161,6 +9161,36 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return varTypeIsStruct(info.compRetNativeType) && (info.compRetBuffArg == BAD_VAR_NUM);
 #endif // TARGET_XXX
 
+#else // not FEATURE_MULTIREG_RET
+
+        // For this architecture there are no multireg returns
+        return false;
+
+#endif // FEATURE_MULTIREG_RET
+    }
+
+    // Returns true if the method returns a value in more than one return register,
+    // it should replace/be  merged with compMethodReturnsMultiRegRetType when #36868 is fixed.
+    bool compMethodReturnsResInMultiplyRegisters()
```

Comment:
Even if temporary, I think it would be good to 1) describe in what way it is different (in particular talk about the special handling required for SIMD types, soon to be additional ones for x64/ux, that are passed in a single register), and 2) rename it to be something like `compMethodReturnsMultiRegRegTypeAlternate`.